grumblrApp/app/controllers/grumblres_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GrumblresController < ApplicationController
+
+  def index
+    @grumblrs = Grumblr.all
+    render status: 200, json: @grumblrs.to_json
+  end
+
+  def show
+    @grumblr = Grumblr.find(params[:id])
+    render status: 200, json: @grumblr.to_json
+  end
+
+  # def create
+  #   @grumblr = Song.new(song_params)
+  #   if @grumblr.save
+  #     render json: @grumblr.to_json, status: 200
+  #   end
+  # end
+  #
+  # def update
+  #   @grumblr = Song.find(params[:id])
+  #   if @grumblr.update(song_params)
+  #     render json: @grumblr.to_json, status: 200
+  #   end
+  # end
+  #
+  # def destroy
+  #   @grumblr = Song.find(params[:id])
+  #   if @grumblr.destroy
+  #     render json: @grumblr.to_json, status: 200
+  #   end
+  # end
+  #
+  # private
+  # def song_params
+  #   params.require(:song).permit(:title, :audio_url, :album_art, :artist, :genre)
+  # end
```

Comment:
i noticed that there some songs spinkled throughout this commented code, i think once you change this to fit the grumbler model is should be fine